common/utils/src/main/resources/error/error-classes.json

```
@@ -2338,6 +2350,12 @@
     ],
     "sqlState" : "42000"
   },
+  "INVALID_VARIABLE_TYPE_FOR_QUERY_EXECUTE_IMMEDIATE" : {
+    "message" : [
+      "Variable passed query must be of type STRING"
```

minor but types must be quoted:
```suggestion
      "Variable type must be a \"STRING\" but got <varType>."
```